landing-page/scripts/batch_api.py

```
@@ -222,21 +221,15 @@ def cantus_bundle(project_id: int, source_id: str, user=Depends(get_current_user
         buf = io.BytesIO()
         used_stems, filenames = set(), []
         with zipfile.ZipFile(buf, "w", zipfile.ZIP_DEFLATED) as zf:
-            for name, xml_content, folio, original_width in rows:
+            for name, xml_content, folio in rows:
                 stem = _sanitize_stem(folio) or _sanitize_stem(name) or "unknown"
                 unique_stem, n = stem, 2
                 while unique_stem in used_stems:
                     unique_stem = f"{stem}-{n}"; n += 1
```

Ruff (0.16.0)

[error] Multiple statements on one line (semicolon)

(E702)